index.css

```diff
@@ -23,43 +23,45 @@ input[type="text"]::-ms-clear {
 
 /* New app menu */
 .app-menu-icon {
-  width: 21px;
-  height: 15px;
+  background: none;
+  border: none;
+  cursor: pointer;
   display: flex;
   flex-direction: column;
+  height: 15px;
   justify-content: space-between;
-  cursor: pointer;
+  padding: 0;
   position: absolute;
-  z-index: 10;
-  transition: all 1s ease;
   top: 16px;
+  transition: all 1s ease;
+  width: 21px;
+  z-index: 10;
 }
 
 @media only screen and (max-width: 768px) {
   .app-menu-icon {
-    margin-left: 15px;
+    left: 15px;
   }
 }
 
-.menu-top-line,
-.menu-middle-line,
-.menu-bottom-line,
-.menu-left-x,
-.menu-right-x {
+.app-menu-icon > span {
   border-bottom: 3px solid #ffffff;
+  display: block;
+  position: relative;
   transition: all 0.5s ease;
+  width: 100%;
 }
 
-.menu-left-x {
+.app-menu-icon.open span:first-child {
```

Comment:
I think I preferred the 3 classes to `nth-child` which can sometimes get messy, but I'll leave it to the second reviewer to decide if this is necessary to change.

Reply:
I replaced the ordinal positioning CSS with the top/middle/bottom nomenclature in  https://github.com/opentripplanner/otp-react-redux/pull/749/commits/3523d0d99f2b12c7b43aadbde6ee57d915966e86 so it is closer from how it was before (and improved).